src/core/src/bootstrap/Constants.py

```diff
@@ -323,3 +323,9 @@ class EnvLayer(EnumBackport):
         NOT_SELECTED: 6
     }
 
+    # Ubuntu Pro Client constants.
+    class UbuntuProClientSettings(EnumBackport):
+        FEATURE_ENABLED = True
+        MINIMUM_PYTHON_VERSION_REQUIRED = (3, 5)
```

Comment:
Do add the reasoning behind why (3,5) is a tuple in a comment.

Reply:
Added comment.